catalog/dags/retired/common/etl/scripts/ExtractCCLinks.py

```diff
@@ -22,7 +22,7 @@
 
 
 BUCKET = os.environ.get("COMMONCRAWL_BUCKET", "not_set")
-
+COMMONSMAPPER_BUCKET = os.environ.get("COMMONSMAPPER_BUCKET", "ov-commonsmapper")
```

Comment:
Won't this cause further issues if `ov-commonsmapper` gets claimed? Or is that a bucket we still control? It might be easier to just leave the default out entirely.